grafana-plugin/src/pages/incident/Incident.tsx

```diff
@@ -126,71 +128,71 @@ class IncidentPage extends React.Component<IncidentPageProps, IncidentPageState>
     }
 
     return (
-      <PageErrorHandlingWrapper errorData={errorData} objectName="alert group" pageName="incidents">
-        {() =>
-          errorData.isNotFoundError ? (
-            <div className={cx('root')}>
-              <div className={cx('not-found')}>
-                <VerticalGroup spacing="lg" align="center">
-                  <Text.Title level={1}>404</Text.Title>
-                  <Text.Title level={4}>Incident not found</Text.Title>
-                  <PluginLink query={{ page: 'incidents', cursor, start, perpage }}>
-                    <Button variant="secondary" icon="arrow-left" size="md">
-                      Go to incidents page
-                    </Button>
-                  </PluginLink>
-                </VerticalGroup>
-              </div>
-            </div>
-          ) : (
-            <>
-              <div className={cx('root')}>
-                {this.renderHeader()}
-                <div className={cx('content')}>
-                  <div className={cx('column')}>
-                    <Incident incident={incident} datetimeReference={this.getIncidentDatetimeReference(incident)} />
-                    <GroupedIncidentsList
-                      id={incident.pk}
-                      getIncidentDatetimeReference={this.getIncidentDatetimeReference}
-                    />
-                    <AttachedIncidentsList id={incident.pk} getUnattachClickHandler={this.getUnattachClickHandler} />
-                  </div>
-                  <div className={cx('column')}>{this.renderTimeline()}</div>
+      <PluginPage pageNav={{ text: '', hideFromBreadcrumbs: true }}>
+        <PageErrorHandlingWrapper errorData={errorData} objectName="alert group" pageName="incidents">
+          {() => (
```

Comment:
it's not related to current PR but just curious why do we need render prop pattern here?

Reply:
It's not necessarily needed, we could also get the job done by using `React.Node` instead, I'm thinking this might come in handy if we add more logic to the wrapper?